components/layouts/masonry.js

```diff
@@ -37,7 +39,10 @@ const Masonry = ({ children }) => {
   }, []);
 
   return (
-    <section className="masonry" style={{ "--max-height": height + "px" }}>
+    <section
+      className={styles.Container}
+      style={{ "--max-height": height + "px" }}
```

Comment:
Is this CSS variable actually used anywhere? I did a quick search on Github but couldn't find it

Reply:
I don't think it is... I deleted it on [64a7579](https://github.com/streamlit/docs/pull/274/commits/64a7579bb583bbaf2138605ec2e881c00703ac7b)